src/components/shapeComponents/ArrayBuilder.vue

```diff
@@ -1,18 +1,21 @@
 <template>
   <div class="array-builder-container">
-    <h3>{{ shape.name }}:</h3>
     <div v-if="loading" class="loading-container">
       <ProgressSpinner />
     </div>
     <div v-else class="children-container" :class="invalid && 'invalid'">
       <small v-if="invalid" class="validate-error">{{ validationErrorMessage }}</small>
-      <template v-for="item of build" :key="item.id">
+      <template v-for="(item, index) in build" :key="item.id">
         <component
           :is="item.type"
           :value="item.value"
           :id="item.id"
           :position="item.position"
-          :showButtons="item.showButtons"
+          :showButtons="
+            index !== build.length - 1
```

Comment:
It looks like this removes the add button if it is not the last item in the list. The array builder shouldn't do this logic here. Some arrays require adding items at different positions. If button needs to be removed, an alternate way will need to be found.